src/components/Auth/Auth.tsx

```diff
@@ -104,11 +143,49 @@ function Auth() {
           <ModeSwitch className="signupLoginText">
             <Typography.Text>{signup ? 'Already have an account?' : "Don't have an account?"}</Typography.Text> <br />
             <Typography.Link onClick={handleModeSwitch}>
-              {signup ? 'Log in' : 'Limited Time First 50 $MAGIC✨ Hour Pod Listeners get 200 FREE credits'}
-            </Typography.Link>
+              {signup ? 'Log in' : 'Limited time First 87 $MAGIC✨ users Free sign up'}
```

Comment:
What happened here? This is a regression

Reply:
i think it happened when i was trying to resolve the conflicts changed it back now, apologies.